### ISSUE TITLE
Option for add_geometry not to reset camera view 

### ISSUE THREAD

**Issue by dkimanius** [isl-org/Open3D#4306]
I know that there is an other closed feature request (#2264) regarding this issue. The suggested solution there is to use the convert_to_pinhole_camera_parameters function to store the camera view and restore it after the call to add_geometry. 

When doing that I noticed this warning when the othogonal view is adjusted:
[Open3D WARNING] [ViewControl] ConvertToPinholeCameraParameters() failed because orthogonal view cannot be translated to a pinhole camera.

One can resolve this issue by reseting the orthogonal view and storing it as a parameter and then restoring also it after the call. However, this is only an issue because add_geometry resets the view in the first place. Why does it do that? And can we add an option (function paramter) to not do that?

**Comment by BhavyanshM** [comment]
I agree. Could we please add an option that prevents `add_geometry` from resetting the view. It would be a very helpful for a lot of users. Thank you so much in advance.

**Comment by mpottinger** [comment]
Also just found out this feature seems to be missing, would be very useful to be able to navigate while geometry is getting updated.

**Comment by schaubli** [comment]
You can use the `reset_bounding_box` parameter (default=True). 
If you set `reset_bounding_box` to `False`, the view point or camera point of view will not change when [adding](http://www.open3d.org/docs/0.9.0/python_api/open3d.visualization.Visualizer.html#open3d.visualization.Visualizer.add_geometry) or [removing](http://www.open3d.org/docs/0.9.0/python_api/open3d.visualization.Visualizer.html#open3d.visualization.Visualizer.remove_geometry) geometry.
